dockerfiles/platform-build/Dockerfile

```diff
@@ -34,12 +34,12 @@ USER hpcc
 
 ARG BUILD_USER=hpcc-systems
 RUN echo BUILD_USER is ${BUILD_USER}
+ARG BUILD_TAG=none
```

Comment:
I know why I moved it, so that it would force the clone to be redone if BUILD_TAG changed,
but I don't remember why that was required/desirable..

A cache-layer with an old clone should be ok, as long a it re-fetches, which it would have done if BUILD_TAG changed before..

So I can't see now why it's necessary, happy to move it back.

Reply:
changed